src-vue/src/views/SettingsView.vue

```diff
@@ -12,6 +12,8 @@
                 <el-button icon="Folder" @click="updateGamePath"/>
             </template>
         </el-input>
+        <h3>About:</h3>
+        UI design inspired by <a target="_blank" href="https://github.com/TFORevive/tforevive_launcher/">TFORevive Launcher</a> (not yet public)
```

Comment:
You should use an element link, for it to have same style as the rest of the application.

```html
<el-link :underline="false" target="_blank" href="https://github.com/TFORevive/tforevive_launcher/" type="primary">TFORevive Launcher</el-link>

```

Reply:
Done with cdf0fc521985cd41e1732038d1c57fd70452fa00